src/app/pages/things/gateways/details/config/gateways.config.component.ts

```diff
@@ -19,16 +19,37 @@ export class GatewaysConfigComponent implements OnInit, OnChanges {
     mqtt_url: '',
     edgex_url: '',
     nats_url: '',
-    export_config: '',
+    export_config: {
```

Comment:
I see many values not initialized. Please, make them optional in the interface and remove them from here.

Reply:
what interface?

Reply:
`MqttConfig`